snowchange.py

```diff
@@ -205,7 +217,7 @@ def create_change_history_table_if_missing(change_history_table, autocommit, ver
   execute_snowflake_query(change_history_table['database_name'], query, autocommit, verbose)
 
 def fetch_change_history(change_history_table, autocommit, verbose):
-  query = 'SELECT VERSION FROM {0}.{1}'.format(change_history_table['schema_name'], change_history_table['table_name'])
+  query = "SELECT VERSION FROM {0}.{1} where SCRIPT_TYPE = 'V' order by INSTALLED_ON desc limit 1".format(change_history_table['schema_name'], change_history_table['table_name'])
```

Comment:
This is not a big deal as the change_history_table won't grow in the magnitude of gbs, but something I have noticed in huge tables in Snowflake. 

```SELECT "VERSION" FROM SNOWCHANGE.CHANGE_HISTORY WHERE INSTALLED_ON = (SELECT MAX (INSTALLED_ON) FROM SNOWCHANGE.CHANGE_HISTORY) ;
  ```
type of query has better performance than

```
SELECT "VERSION" FROM  SNOWCHANGE.CHANGE_HISTORY ORDER BY INSTALLED_ON DESC LIMIT 1;
```
May be because Snowflake caches MAX and MIN statistics of the columns ?